.changeset/beige-planes-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sumup/circuit-ui": patch
+---
+
+Fix event and values handling for the experimental ColorInput component.
```

Comment:
By convention, changesets should use past tense.

```suggestion
Fixed event and values handling for the experimental ColorInput component.
```

Reply:
Thanks, fixed.